packages/ui-extensions/src/surfaces/point-of-sale/components.ts

```diff
@@ -1,29 +1,34 @@
 export {Badge} from './components/Badge/Badge';
-export type {BadgeProps} from './components/Badge/Badge';
+export type {BadgeProps, BadgeVariant} from './components/Badge/Badge';
```

Comment:
@agorovyi That was a solid suggestion. Exporting the types from the point-of-sale.ts file exposed all the types that were not being exported from the ui-extensions package, so we caught a lot of potential errors haha.